### DISCUSSION TITLE
Is it necessary to add a sub-module named "dp view" to present graphical results?

### DISCUSSION THREAD

**Discussion by panxiang126** [deepmodeling/deepmd-kit#2758] [Ideas]
As the title suggests, I think this is a non-essential but user experience enhancing idea.

**Comment by panxiang126** [comment]
Concretely, `matplotlib` is called to plot the changes in key parameters after train, the error in energy & force and the distribution of force after test, etc.

**Comment by njzjz** [comment]
I prefer implementing this feature in a separate project.
